Lab6.ipynb

```diff
@@ -0,0 +1,443 @@
+      "text/plain": [
+       "<Figure size 1296x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "a1 = stats.poisson.rvs(100, size = 10000)\n",
+    "b1 = np.array([])\n",
+    "signal = 250\n",
+    "for i in a1:\n",
+    "    b1 = np.append(b1, i + signal)\n",
+    "    \n",
+    "plt.hist(b1, bins = 200)\n",
+    "plt.title('Signal')\n",
+    "plt.xlabel('Measurement')\n",
+    "plt.ylabel('Counts')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see a Poisson distribution with a mean of 350, which is what we had before."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Problem 3c"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we reverse, so we want to know the what the original injected signal was from the observed one. We saw from previous questions that they only were offset by the mean, so we can subtrack the background and be left with the signal. For this example we will use mean of 400."
```

Comment:
Problem 3C:
- typo in explanation: subtrack --: subtract
- Subtracting the mean of the background makes sense.
- Comparison graph makes sense and looks good, clearly shows they are not centered at the same value. You can emphasize that in your explanation too, similar to HW 5.